.gitignore

```diff
@@ -8,6 +8,9 @@ out/
 !locales/**/out/  
 package.nls.*.json
 
+# virtual environment
+env/
```

Comment:
What are your thoughts on `venv/` vs `env/`

Reply:
It doesn't really matter! If you prefer venv/ as the folder, I can change it.

Reply:
changed!